### PR TITLE
Add support for connect to Yandex.Cloud ClickHouse

### PR DESCRIPTION
**Pull request by Negasus**
Use authorization headers for connect to Yandex.Cloud Managed ClickHouse

[Documentation](https://cloud.yandex.com/docs/managed-clickhouse/)

### CODE REVIEW

File: src/partials/config.html [hunk 1]
```diff
@@ -1,31 +1,58 @@
...
-<div class="gf-form-inline">
-    <gf-form-switch class="gf-form max-width-30"
-                    ng-model='ctrl.current.jsonData.addCorsHeader'
-                    label="Add CORS flag to requests" tooltip="Whether 'add_http_cors_header=1' parameter should be attached to requests. Remember that read-only users cannot override this setting."
-                    checked="ctrl.current.jsonData.addCorsHeader" label-class="width-25" switch-class="max-width-5">
-    </gf-form-switch>
+        <div class="gf-form-inline">
+            <div class="gf-form max-width-30">
+                <span class="gf-form-label width-10">X-ClickHouse-Key</span>
+                <input type="password" class="gf-form-input max-width-20" ng-model='ctrl.current.jsonData.xHeaderKey' placeholder="none"></input>
```
Comment (hagen1778) [inline src/partials/config.html]: I'd suggest following placeholder:
`DB user password`

File: src/partials/config.html [hunk 2]
```diff
@@ -1,31 +1,58 @@
...
+                        label="Use Yandex.Cloud authorization headers" tooltip="Use authorization headers for managed Yandex.Cloud ClickHouse database"
+                        checked="ctrl.current.jsonData.useYandexCloudAuthorization" label-class="width-25" switch-class="max-width-5">
+        </gf-form-switch>
+    </div>
+    
+    <div ng-if="ctrl.current.jsonData.useYandexCloudAuthorization">
+        <div class="gf-form-inline">
+            <div class="gf-form max-width-30">
+                <span class="gf-form-label width-10">X-ClickHouse-User</span>
+                <input type="text" class="gf-form-input max-width-20" ng-model='ctrl.current.jsonData.xHeaderUser' placeholder="none"></input>
```
Comment (hagen1778) [inline src/partials/config.html]: I'd suggest following placeholder:
`DB user name`

**Comment by Negasus** [comment]
Updated

**Comment by hagen1778** [comment]
Great! Thank you!

**Pull request merged**